linkml_runtime/utils/schemaview.py

```diff
@@ -802,18 +835,6 @@ def permissible_value_children(
         """:param enum_name: parent enum name
         :param permissible_value: permissible value
         :return: all direct child permissible values (is_a)
-
-        CAT:
-        LION:
-          is_a: CAT
```

Comment:
these are from some tests -- not sure how they got into the documentation of the SchemaView itself!